Lavanya/HTMLassignment3.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+	<title>HTMLassignment3</title>
+	<meta charset="UTF-8">
+</head>
+<body>
+
+	<h2>Register</h2>
+
+	<form id="RegistrationForm">
+		<label for="First Name">First Name</label>
+		<input type="text" name="First Name"required/>
+		<label for="Last Name">Last Name</label>
+		<input type="text" name="lastname"required>
+		<br><br>
+		Gender:
+		<label> Male 
+                	<input type="radio" name="gender" id="male" value="male" checked required/> 
+		</label>
+   		<label> Female
+			<input type="radio" name="gender" id="female" value="female">
+		</label> 
+		<label> Others
+			<input type="radio" name="gender" id="others" value="others">
+		</label>

```

Comment:
Input tag should be outside the label tag.

Reply:
Also point label to input tag using "for" property, so that your code is compatible with screen readers and is more accessible.